src/dataDisplay/tableCell.jsx

```diff
@@ -1,14 +1,66 @@
-'use strict';
-
-import React, { Component } from 'react';
+import React from 'react';
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
-import Utils from '../utils/utils.js';
+import Utils from '../utils/utils';
+
+const columnNumberEnums = ['auto', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+const fontSizeEnums = ['large', 'medium', 'small', 'xsmall'];
+const textAlignEnums = ['center', 'left', 'right'];
+const verticalAlignEnums = ['bottom', 'middle', 'top'];
+
+const propTypes = {
+    active: PropTypes.bool,
+    children: PropTypes.node,
+    className: PropTypes.string,
+    collapsing: PropTypes.bool,
+    desktop: PropTypes.oneOfType([
+        PropTypes.bool,
+        PropTypes.oneOf(columnNumberEnums),
+    ]),
+    desktopLarge: PropTypes.oneOfType([
+        PropTypes.bool,
+        PropTypes.oneOf(columnNumberEnums),
+    ]),
+    disabled: PropTypes.bool,
```

Comment:
In regards to issue #25, please add a new prop, `disable`, and do the work below to make disabled secondary, making sure to not regress anything please.

Reply:
This is being resolved in #163.